### PR TITLE
fix minor bug with new RNG parameter, fix issue with transformer not …

### PR DESCRIPTION
**Pull request by pazzo83**
…being exported

Noticed the changes I made actually ended up with model not being exported from package.  Also found a small bug in the way I was importing "Random" into the package.

### CODE REVIEW

File: src/MLJTSVDInterface.jl [hunk 1]
```diff
@@ -19,15 +19,12 @@ decomposition (SVD). Contrary to PCA, this estimator does not center the data be
...
 
 """
-mutable struct TSVDTransformer <: MLJModelInterface.Unsupervised
-    nvals::Int
-    maxiter::Int
-    rng::Union{Int, AbstractRNG}
+MMI.@mlj_model mutable struct TSVDTransformer <: MLJModelInterface.Unsupervised
+    nvals::Int = 2
+    maxiter::Int = 1000
+    rng::Untion{Int, AbstractRNG} = GLOBAL_RNG
```
Comment (ablaom) [inline src/MLJTSVDInterface.jl]: typo: Untion

**Review by ablaom — COMMENTED** [review]
Looks good to me although I don't understand how tests are passing with that typo!

File: src/MLJTSVDInterface.jl (hunk 1 above)
Comment (pazzo83) [reply src/MLJTSVDInterface.jl]: lol thanks

Comment (pazzo83) [reply src/MLJTSVDInterface.jl]: looks like that is fixed in the most recent commit

**Comment by ablaom** [comment]
@pazzo83 Would you be so kind as to bump the version number if necessary. Make a PR dev -> master, and tag a new release using the pinned issue created for that purpose? https://github.com/JuliaAI/MLJTSVDInterface.jl/issues/7#issuecomment-968357389

**Pull request merged**